examples/nlp/glue_benchmark/glue_benchmark_config.yaml

```diff
@@ -29,18 +29,17 @@ model:
     drop_last: false
 
   train_ds:
-    file_name: 'train.tsv'
+    ds_item: 'train.tsv'
```

Comment:
hmmm, what ds stands for? dataset? what was wrong with file_name?

Reply:
@ekmb Train_ds does not support multi dataset, so name ds_item isn't treated differently. Probably best to keep it as file_name

Reply:
@titu1994 I don't need multi dataset for training, just wanted to be consistent with validation. 
Discussed with @tkornuta-nvidia offline.